--- dwmstatus.c	2020-09-24 19:49:27.607050883 +0200
+++ dwmstatus.c.new	2020-09-24 19:38:49.507045474 +0200
@@ -179,43 +179,20 @@
 main(void)
 {
 	char *status;
-	char *avgs;
-	char *bat;
-	char *bat1;
-	char *tmar;
-	char *tmutc;
 	char *tmbln;
-	char *t0, *t1, *t2;
-
+	
 	if (!(dpy = XOpenDisplay(NULL))) {
 		fprintf(stderr, "dwmstatus: cannot open display.\n");
 		return 1;
 	}
 
 	for (;;sleep(60)) {
-		avgs = loadavg();
-		bat = getbattery("/sys/class/power_supply/BAT0");
-		bat1 = getbattery("/sys/class/power_supply/BAT1");
-		tmar = mktimes("%H:%M", tzargentina);
-		tmutc = mktimes("%H:%M", tzutc);
 		tmbln = mktimes("KW %W %a %d %b %H:%M %Z %Y", tzberlin);
-		t0 = gettemperature("/sys/devices/virtual/hwmon/hwmon0", "temp1_input");
-		t1 = gettemperature("/sys/devices/virtual/hwmon/hwmon2", "temp1_input");
-		t2 = gettemperature("/sys/devices/virtual/hwmon/hwmon4", "temp1_input");
 
-		status = smprintf("T:%s|%s|%s L:%s B:%s|%s A:%s U:%s %s",
-				t0, t1, t2, avgs, bat, bat1, tmar, tmutc,
+		status = smprintf("Misterbzr | %s",				
 				tmbln);
 		setstatus(status);
 
-		free(t0);
-		free(t1);
-		free(t2);
-		free(avgs);
-		free(bat);
-		free(bat1);
-		free(tmar);
-		free(tmutc);
 		free(tmbln);
 		free(status);
 	}
